tasks/prepare-exodus-params/prepare-exodus-params.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: prepare-exodus-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that prepares exodus configuration options from the data file
+  params:
+    - name: dataPath
+      type: string
+      description: Path to the JSON string of the merged data containing the exodus configuration options to use
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  results:
+    - name: exodusGwUrl
+      type: string
+      description: |
+        The URL of the Exodus Gateway to use
+    - name: exodusGwEnv
+      type: string
+      description: |
+        The environment of the Exodus Gateway to use
+    - name: exodusGwCert
+      type: string
+      description: |
+        The certificate to use to authenticate to exodus-gw
+    - name: exodusGwKey
+      type: string
+      description: |
+        The key to use to authenticate to exodus-gw
+  steps:
+    - name: collect-exodus-params
+      image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+      script: |
+        #!/usr/bin/env sh
+        set -x
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        if [ $(jq '.exodus | has("env")' $DATA_FILE) == false ] ; then
+            echo "Exodus env missing in data JSON file"
+            exit 1
+        fi
+
+        EXODUS_ENV=$(jq -r '.exodus.env' $DATA_FILE)
+        EXODUS_GW_URL="https://exodus-gw.corp.redhat.com"
+        EXODUS_GW_ENV="live"
+        EXODUS_GW_CERT="exodus-prod-cert.crt"
+        EXODUS_GW_KEY="exodus-prod-privatekey.pem"
```

Comment:
Same comment as above. But also, do we really want to use the encrypted pem key here? Because typically that one is protected by a password, no? I would think the secret should contain the unencrypted key.

Reply:
ack, updated to exodusGwSecret instead